playground/src/app.js

```diff
@@ -125,15 +121,13 @@ events.subscribe(EVENT_SET_RUNTIME, (newRuntime) => {
   }
   validator.validate(editor.getSource());
   activeRuntime = newRuntime;
-
-  const emailButton = document.getElementById('import-email');
-  if (emailButton) {
-    emailButton.classList.toggle('hidden', activeRuntime.id !== 'amp4email');
-  }
 });
 
 runtimes.init();
 
+EmailLoader.createEmailLoader(editor);
```

Comment:
The email loaders only dependency to `editor` is by using `editor.setSource`. You could decouple this by using `Editor.EVENT_UPDATE_EDITOR_CONTENT`.

This would also allow moving the creation of `EmailLoader` to the importer and calling its methods.